mainapp/templates/nav.html

```diff
@@ -8,6 +8,14 @@
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
         <link rel="stylesheet" type="text/css" href="{% static 'styles.css' %}">
         <title>NYC CityStreets</title>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+        <script>
+        $(document).ready(function(){
+          $("#complaints_button").click(function(){
+            $("#complaints").toggle();
```

Comment:
Love this functionality!

Reply:
Good idea! Added script blocks to `nav.html`.